prepareComments.js

```diff
@@ -0,0 +1,61 @@
+
+fs = require('fs');
+const data = fs.readFileSync('eslint.json', 'utf8');
+const myArgs = process.argv.slice(2);
+commitId = myArgs[0]
+
+function main(){
+    prepareComment(data);
+}
+
+function createComments(comPath, comCommitId, comLine, comBody, fileName) {
+    console.log("IN FUNCTION -->>"  + comLine );
+    let com = {
```

Comment:
Expected indentation of 2 spaces but found 4.
'com' is never reassigned. Use 'const' instead.